cypress/integration/accordions.spec.js

```diff
@@ -66,13 +66,13 @@ describe('Accordion Component', () => {
             .should('not.exist');
     });
 
-    it('should show a down arrow icon when the accordion is expaned', () => {
+    it('should show a right arrow icon when the accordion is expaned', () => {
```

Comment:
Should this be 'should show a right arrow icon when the accordion is _**collapsed**_'